components/count-badge/count-badge.js

```diff
@@ -134,7 +164,11 @@ class CountBadge extends RtlMixin(LitElement) {
 			numberString = `${'9'.repeat(this.maxDigits)}+`;
 		}
 		return html`
-        	<div class="d2l-count-badge-number">${numberString}</div>`;
+        	<div class="d2l-count-badge-number" 
+				tabindex="${this.tabStop ? '0' : '-1'}" 
+				role="${this.announceChanges ? 'status' : 'presentation' }"
```

Comment:
Hmm, `presentation` things don't end up as anything semantic in the accessibility API, which I'm not sure we want. Maybe just either `status` or leave it undefined and use the `ifDefined` directive? That way it just won't set the attribute at all.

Reply:
Yep, leaving out the role if we don't need it makes sense. I'm still playing around between using role and the aria-live attribute, both are failing the axe tests currently :(